src/main/resources/config/init.xml

```diff
@@ -47,6 +47,7 @@
 	<parameter name="privateKeyFilePath" description="path to private key file"/>
 	<parameter name="privateKeyPassword" description="Passphrase of the private key"/>
 	<parameter name="setAvoidPermission" description="Sets whether to avoid file permission check."/>
+	<parameter name="sftpPoolConnectionAgedTimeout" description="Interval to close connections in the connection pool in seconds"/>
```

Comment:
Shall we add this to the UISchema file as well?
https://github.com/wso2-extensions/esb-connector-file/blob/master/src/main/resources/uischema/sftp.json